src/App.js

```diff
@@ -1,9 +1,106 @@
+import { useState } from "react";
+import './style.css'; //이번 과제는 컴포넌트 분리가 어려워서 css를 사용했습니다 ㅠㅠ
+import './background.js';
+import Clock from './clock.js';
 
 function App() {
+  const [toDo, inputToDo] = useState("");
+  const [toDos, setToDos] = useState([]);
+  const [doneToDos, movetoDos] = useState([]); 
```

Comment:
setter의 이름은 보통 아래와 같이 `set + State` 형태로 짓습니다.

```suggestion
  const [toDo, setToDo] = useState("");
  const [toDoList, setToDoList] = useState([]);
  const [doneToDoList, setDoneToDoList] = useState([]); 
```

Reply:
useState에 사용하는 네이밍 패턴에 대해서는 저도 처음에 많이 헷갈렸는데요,
(물론 아직도 헷갈립니다)
세은님께서 작성하신 코드들을 보면 기본적인 감각이 있으셔서
React에 더 익숙해지실수록 지금보다도 좋은 네이밍을 작성해 주실 것 같아 기대됩니다!